### DISCUSSION TITLE
Image Upload in Tag-Album leads to 404 Error

### DISCUSSION THREAD

**Discussion by yu-tendo** [LycheeOrg/Lychee#2499] [Bugs] [answered]
### Lychee version

v5.5.1

### Which PHP version are you using?

PHP 8.3

### Detailed description of the problem

I just installed Lychee v5. Everything seems to work fine just not the image upload in a "Tag-Album" -it works in a regular album or when I directly try to upload one on the main page.

### Steps to reproduce the issue

1. Click on the + symbol (top-right corner)
2. Create a new "Tag Album"
3. Inside the new "Tag Album", click again on the + symbol (top-right corner) and select "Upload Foto"
4. Error message appears (No query results for model [App\Models\Album] fr2xPE0VDJ3oRXCMfmZYqiD3)

### Diagnostics [REQUIRED]

[lychee@canvas ~]$ php artisan lychee:diagnostics

Diagnostics
-----------
Warning: Dropbox import not working. dropbox_key is empty.
Warning: You may experience problems when uploading a photo of large size. Take a look in the FAQ for details.
Warning: You may experience problems when uploading a photo of large size. Take a look in the FAQ for details.
Error: /o************it (and subdirectories) are not executable, check the permissions

System Information
------------------
Lychee Version (tag):                    v5.5.1 (656ffae) -- Up to date (-0.00047074450016444 years ago).
DB Version:                              5.5.1

composer install:                        --no-dev
APP_ENV:                                 production
APP_DEBUG:                               false
APP_URL:                                 set
APP_DIR:                                 default
LOG_VIEWER_ENABLED:                      true
LIVEWIRE_ENABLED:                        true

System:                                  Linux
PHP Version:                             8.3.9
PHP User agent:                          Lychee/4 (https://lycheeorg.github.io/)
Timezone:                                Europe/Berlin
Max uploaded file size:                  2M
Max post size:                           8M
Livewire chunk size:                     2.00 MB
Max execution time:                      0
MySQL Version:                           11.4.2-MariaDB

exec() Available:                        yes
Imagick Available:                       1
Imagick Enabled:                         1
Imagick Version:                         1809
GD Version:                              2.3.3
Number of foreign key:                   12 found.

Config Information
------------------
version:                                 050501
check_for_updates:                       0
sorting_photos_col:                      title
sorting_photos_order:                    ASC
sorting_albums_col:                      created_at
sorting_albums_order:                    ASC
imagick:                                 1
skip_duplicates:                         0
small_max_width:                         0
small_max_height:                        360
medium_max_width:                        1920
medium_max_height:                       1080
lang:                                    en
image_overlay_type:                      none
default_license:                         none
compression_quality:                     90
grants_full_photo_access:                1
delete_imported:                         0
mod_frame_enabled:                       1
mod_frame_refresh:                       30
thumb_2x:                                1
small_2x:                                1
medium_2x:                               1
landing_page_enable:                     0
force_migration_in_production:           0
search_public:                           0
hide_version_number:                     0
SL_enable:                               0
SL_for_admin:                            0
recent_age:                              1
grants_download:                         0
photos_wraparound:                       1
map_display:                             0
zip64:                                   1
map_display_public:                      0
map_provider:                            Wikimedia
force_32bit_ids:                         0
map_include_subalbums:                   0
update_check_every_days:                 3
has_exiftool:                            0
share_button_visible:                    0
import_via_symlink:                      0
has_ffmpeg:                              0
location_decoding:                       0
location_decoding_timeout:               30
location_show:                           1
location_show_public:                    0
rss_enable:                              0
rss_recent_days:                         7
rss_max_items:                           100
prefer_available_xmp_metadata:           0
editor_enabled:                          1
lossless_optimization:                   0
swipe_tolerance_x:                       150
swipe_tolerance_y:                       250
local_takestamp_video_formats:           .avi|.mov
log_max_num_line:                        1000
unlock_password_photos_with_url_param:   0
nsfw_visible:                            1
nsfw_blur:                               0
nsfw_warning:                            0
nsfw_warning_admin:                      0
nsfw_banner_override:
map_display_direction:                   1
album_subtitle_type:                     oldstyle
upload_processing_limit:                 4
new_photos_notification:                 0
legacy_id_redirection:                   1
zip_deflate_level:                       6
default_album_protection:                1
album_decoration:                        layers
album_decoration_orientation:            row
allow_username_change:                   1
auto_fix_orientation:                    1
use_job_queues:                          0
random_album_id:                         starred
use_last_modified_date_when_no_exif_date: 0
layout:                                  justified
date_format_photo_thumb:                 M j, Y, g:i:s A e
date_format_photo_overlay:               M j, Y, g:i:s A e
date_format_sidebar_uploaded:            M j, Y, g:i:s A e
date_format_sidebar_taken_at:            M j, Y, g:i:s A e
date_format_hero_min_max:                F Y
date_format_hero_created_at:             M j, Y, g:i:s A T
date_format_album_thumb:                 M Y
upload_chunk_size:                       0
nsfw_banner_blur_backdrop:               0
search_pagination_limit:                 1000
search_minimum_length_required:          4
photo_layout_justified_row_height:       320
photo_layout_masonry_column_width:       300
photo_layout_grid_column_width:          250
photo_layout_square_column_width:        200
photo_layout_gap:                        12
display_thumb_album_overlay:             always
display_thumb_photo_overlay:             hover
default_album_thumb_aspect_ratio:        1/1
use_album_compact_header:                0
login_button_position:                   left
back_button_enabled:                     0
back_button_text:                        Return to Home
current_job_processing_visible:          1
SA_random_thumbs:                        0
job_processing_queue_visible:            1
login_required:                          0
enable_unsorted:                         1
enable_starred:                          1
enable_recent:                           1
enable_on_this_day:                      1
login_required_root_only:                1

### Browser & System [REQUIRED]

Browser: Mozilla Firefox 127.0.2 (64-bit)
OS: Windows 10 64-bit

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by ildyria** [comment, marked as answer]
Working as intended.
You are not supposed to upload in tag albums.

Those should be seen as "smart albums" (even though it is possible to upload in `unsorted` and a few other.

**Reply by yu-tendo** [reply]
Thanks for the quick reply! If this is intended would it be possible to hide or disable the upload image option in a smart/tag album?

**Reply by ildyria** [reply]
In v6 yes.
I completely stop dev in the livewire front-end to focus entirely on the new one.
See more here: https://github.com/LycheeOrg/Lychee/discussions/2485

**Reply by yu-tendo** [reply]
I understand -thanks for the explanation!
